google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -204,6 +207,81 @@ public void testBatchWriteWithCommittedStream()
     assertEquals(false, iter.hasNext());
   }
 
+  @Test
+  public void testJsonStreamWriterBatchWriteWithCommittedStream()
```

Comment:
One e2e test about schema update handling?

Reply:
Got it.